pyproject.toml

```diff
@@ -39,7 +39,7 @@ PyYAML = "*"
 pylink-square = "^0.12.0"
 pyserial = "*"
 python-can = {extras = ["pcan"], version = "^4.0.0"}
-python-uds = {git = "https://github.com/BKaDamien/python-uds.git", rev = "2.0.2"}
+python-uds = {git = "https://github.com/sebclrsn/python-uds.git", branch = "feature/enable-server-side-isotp"}
```

Comment:
We now have the kiso-testing-python-uds, the changes should be put there

Reply:
Updated to pykiso-python-uds ^2.1.0 ;)